homeassistant/components/heos/media_player.py

```diff
@@ -107,6 +120,7 @@ async def _player_update(self, player_id, event):
     async def _heos_updated(self):
         """Handle sources changed."""
         await self.async_update_ha_state(True)
+        self._group = await self.async_get_group()
```

Comment:
This is redundant as `async_update_ha_state(True)` will call `async_update()` and that already performs this logic.